care/emr/resources/scheduling/schedule/spec.py

```diff
@@ -77,6 +78,12 @@ def validate_for_slot_type(self):
                 end_time = datetime.datetime.combine(
                     datetime.datetime.now(tz=UTC).date(), availability.end_time
                 )
+                total_time = (end_time - start_time).total_seconds() / 60
+                total_slots = (
+                    total_time / self.slot_size_in_minutes
+                ) * self.tokens_per_slot
+                if total_slots > settings.MAX_SLOTS_PER_AVAILABILITY:
+                    raise ValueError("Daily slot allocation limit exceeded")
```

Comment:
The error message could be friendlier by letting know the client about the max limit as well. Client would have no idea what the limit would be with this error message